x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/existence/FrozenExistenceDeciderIT.java

```diff
@@ -131,7 +131,7 @@ public void testZeroToOne() throws Exception {
             String[] indices = indices();
             assertThat(indices, arrayContaining(PARTIAL_INDEX_NAME));
             assertThat(indices, not(arrayContaining(INDEX_NAME)));
```

Comment:
Currently this is failing with:
```
java.lang.AssertionError: 	
Expected: ["partial-index"] 	
     but: item 0: was "index"	
```

It would be great to know if it is not created `partial-index` yet or both indices exists at the same time (this is an odd situation).

I wish we have assertj here so that it would report entire array in case of failure.
I have tried `assertThat(indices, allOf(arrayContaining(PARTIAL_INDEX_NAME), not(arrayContaining(INDEX_NAME))));` but its error output is also not clear